protocols/imap/src/test/java/org/apache/james/imap/processor/base/UidMsnMapperTest.java

```diff
@@ -122,7 +122,7 @@ public void isEmptyShouldReturnTrueIfNoMapping() {
     }
 
     @Test
-    public void isEmptyShouldReturnFalseIfNoMapping() {
+    public void isEmptyShouldReturnFalseIfMapping() {
```

Comment:
I don't understand why you rename the test, also I don't understand what the test really test..